src/style.css

```diff
@@ -0,0 +1,84 @@
+/* stylelint-disable max-line-length */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+
+  --color-white: #fff;
+  --color-silver: #e1e7ed;
+  --color-gray: #3d4e61;
+  --color-dark: rgba(61, 78, 97, 0.1);
```

Comment:
move it to :root selector
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#basic_usage

Reply:
Ok, its easy)